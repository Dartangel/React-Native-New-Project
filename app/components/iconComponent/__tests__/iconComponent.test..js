import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import IconComponent from '../index';

describe('IconComponent', () => {
    test('IconComponent snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<IconComponent />);
        expect(snapshot).toMatchSnapshot();
    });
});