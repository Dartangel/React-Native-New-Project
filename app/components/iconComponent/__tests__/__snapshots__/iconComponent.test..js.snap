// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconComponent IconComponent snapshot 1`] = `
<Icon
  allowFontScaling={false}
  color="#FFF"
  size={25}
/>
`;
